src/ledger/LedgerTxnImpl.h

```diff
@@ -584,6 +584,8 @@ class LedgerTxn::Impl
 
     double getPrefetchHitRate() const;
 
+    bool HasSponsorshipEntry() const;
```

Comment:
Can you add a comment about the exception safety guarantees? Feel free to ask me if you have questions about this.

Reply:
Done